contrib/check-config.sh

```diff
@@ -181,6 +181,9 @@ echo
 
 echo 'Optional Features:'
 {
+	check_flags USER_NS
+}
+{
```

Comment:
There should be no need of this new block right? you can append `USER_NS` at the end of https://github.com/docker/docker/pull/16914/files#diff-a7d4f99f6b6a9d7855588d5d2963f8a2L184